tasks/package.yaml

```diff
@@ -9,9 +9,9 @@
     - sudo_pkg
 
 - name: Install sudo for RedHat
-  yum:
-    name: sudo
-    state: present
+  action:
+    "{{ ansible_pkg_mgr }} pkg=sudo
+    state=present"
```

Comment:
np: you're the author, it's your call.
I'll be using my fork till I switch to 2.0